libcudacxx/codegen/codegen.cpp

```diff
@@ -178,6 +181,7 @@ int main() {
             for(auto& cv: cv_qualifier) {
                 out << "template<class _Type, _CUDA_VSTD::__enable_if_t<sizeof(_Type)==" << sz/8 << ", int> = 0>\n";
                 out << "_LIBCUDACXX_DEVICE void __atomic_store_cuda(" << cv << "_Type *__ptr, _Type *__val, int __memorder, " << scopenametag(s.first) << ") {\n";
+		out << "    if (__cuda_store_weak_if_local(__ptr, *__val)) return;\n";
```

Comment:
Swap tabs for spaces throughout this file.

Reply:
```suggestion
                out << "    if (__cuda_store_weak_if_local(__ptr, *__val)) return;\n";
```